java/src/apps/AppsBundle_de.properties

```diff
@@ -27,13 +25,13 @@ MenuRoster = Lokpark
 MenuOperations = Betrieb
 
 MenuHelp = Hilfe
-MenuItemWindowHelp = Fenster Hilfe...
+MenuItemWindowHelp = Fenster Hilfe
```

Comment:
I don't think the ellipsis should be retained in English or German as the help content is not a dialog, but a window.